ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -22,10 +25,15 @@ export interface ListingCardProps {
   tableHeaderProps?: ListingCardHeaderProps
   tableProps: ListingCardTableProps
   detailsLinkClass?: string
+  listingID: string
+  addFavorite: (arg0: string) => void
```

Comment:
Do we have a usecase where we will want to override the addFavorite/removeFavorite functions? If not I think it's simpler to just import the add/remove functions directly here